run_integration_test.sh

```diff
@@ -50,10 +50,10 @@ tar -xzOf ./v1.tar.gz bundle.tar | tar -xf - --directory ./v1_bundle
 # Run tests
 if [[ ${ROS_DISTRO} = "kinetic" ]]
 then
-	docker run -it -v $(pwd):/workspace ubuntu:xenial /workspace/test_v1.sh
-	docker run -it -v $(pwd):/workspace ubuntu:xenial /workspace/test_v2.sh
+	docker run -v $(pwd):/workspace ubuntu:xenial /workspace/test_v1.sh
+	docker run -v $(pwd):/workspace ubuntu:xenial /workspace/test_v2.sh
 else
-	docker run -it -v $(pwd):/workspace ubuntu:bionic /workspace/test_v1.sh
-	docker run -it -v $(pwd):/workspace ubuntu:bionic /workspace/test_v2.sh
+	docker run -v $(pwd):/workspace ubuntu:bionic /workspace/test_v1.sh
+	docker run -v $(pwd):/workspace ubuntu:bionic /workspace/test_v2.sh
```

Comment:
Make sure this won't break travis.

Reply:
The [Travis build passes](https://travis-ci.org/colcon/colcon-bundle/builds/643178993?utm_source=github_status&utm_medium=notification) so we're good on that end.
In any case, we're leaving the Travis config so lets see how GH actions compare to Travis over the next few weeks.